testgrid/config.yaml

```diff
@@ -4790,6 +4790,10 @@ dashboards:
     test_group_name: ci-kubernetes-e2e-kubeadm-gce-dns-coredns
     alert_options:
       alert_mail_to_addresses: 'kubernetes-sig-network-test-failures@googlegroups.com'
+  - name: ingress-gce-e2e-containerd
```

Comment:
Can you rename this to containerd-e2e-gci-gce-ingress

Reply:
Will do.